docker/docker.go

```diff
@@ -1,74 +1,80 @@
 package docker
 
 import (
-	"github.com/alaa/pencil-go/registry"
+	"fmt"
+	"github.com/alaa/pencil-go/container"
 	docker "github.com/fsouza/go-dockerclient"
```

Comment:
It might be better to call this `client` or `dockerClient`

I was a little confused lower down because I thought `docker.NewClient` was this packer `docker`